playwright/e2e/regression-tests/pills-click-in-app.spec.ts

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { test, expect } from "../../element-web-test";
+
+test.describe("Pills", () => {
+    test.use({
+        displayName: "Sally",
+    });
+
+    test("should navigate clicks internally to the app", async ({ page, app, user }) => {
+        const messageRoom = "Send Messages Here";
+        const targetLocalpart = "aliasssssssssssss";
+        await app.client.createRoom({
+            name: "Target",
+            room_alias_name: targetLocalpart,
+        });
+        const messageRoomId = await app.client.createRoom({
+            name: messageRoom,
+        });
+
+        // discard the target room ID - we don't need it
+        await app.viewRoomByName(messageRoom);
+        await expect(page).toHaveURL(new RegExp(`\\/#\\/room\\/${messageRoomId}`));
```

Comment:
No need to escape `/` when using `new RegExp`
```suggestion
        await expect(page).toHaveURL(new RegExp(`/#/room/${messageRoomId}`));
```